scene/gui/texture_progress_bar.h

```diff
@@ -54,7 +54,8 @@ class TextureProgressBar : public Range {
 		FILL_COUNTER_CLOCKWISE,
 		FILL_BILINEAR_LEFT_AND_RIGHT,
 		FILL_BILINEAR_TOP_AND_BOTTOM,
-		FILL_CLOCKWISE_AND_COUNTER_CLOCKWISE
+		FILL_CLOCKWISE_AND_COUNTER_CLOCKWISE,
+		FILL_MODE_MAX
```

Comment:
Add a trailing comma since the enum is written over several lines:

```suggestion
		FILL_CLOCKWISE_AND_COUNTER_CLOCKWISE,
		FILL_MODE_MAX,
```

This is likely something we'll do through the whole codebase in the future, but it doesn't hurt to do it now :slightly_smiling_face:

Reply:
Done.